### PR TITLE
ceph: prometheus rules format changes

### PR DESCRIPTION
**Pull request by aruniiird**
Due to the latest jsonnet and jsonnetfmt, v0.17.0, we have the
`description` and `message` lines format changed.

Signed-off-by: Arun Kumar Mohan <amohan@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml [hunk 1]
```diff
@@ -350,4 +326,3 @@ spec:
       for: 1m
       labels:
         severity: critical
-
```
Comment (leseb) [inline cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml:353]: keep the new line? or is it auto-generated?

**Review by leseb — CHANGES_REQUESTED** [review]
https://github.com/rook/rook/pull/8774#discussion_r713696826

File: cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml (hunk 1 above)
Comment (aruniiird) [reply cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml:353]: that should be a copy-paste mistake. added the last newline

Comment (aruniiird) [reply cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml]: added the newline...

**Review by leseb — APPROVED** [review]

**Comment by leseb** [comment]
@aruniiird does this need to be backported?

**Comment by aruniiird** [comment]
> @aruniiird does this need to be backported?

Yes @leseb ...

**Pull request merged**